PRINCIPLES.md

```diff
@@ -12,7 +12,15 @@ Last reviewed and concurred: Jul 27, 2022, [Issue 413](https://github.com/cncf/t
 [No Kingmakers & One Size Does Not Fit All](#no-kingmakers--one-size-does-not-fit-all)  
 [Not a Standards Body](#not-a-standards-body)  
 [We Want a Comprehensive Toolchain](#we-want-a-comprehensive-toolchain)    
-[Above All We Want To Help Projects](#above-all-we-want-to-help-projects)  
+[Above All We Want To Help Projects](#above-all-we-want-to-help-projects)
+
+
+[Technical Leadership Principles](#technical-leadership-principles)    
+[Purpose](#purpose)    
+[Be welcoming](#be-welcoming)    
+[Always do better](#always-do-better)    
+[Foster respectful resolution](#foster-respectful-resolution)    
+[Facilitate open, transparent, and inclusive decision-making](#facilitate-open-transparent-and-inclusive-decision-making)
```

Comment:
nit: as this is a table of contents, could you indent these lines or make them a bulleted list in Markdown?  This would also be useful to do to the section above.

Reply:
https://github.com/cncf/toc/pull/1265